### PR TITLE
Handle unpublishable packages missing a version

### PR DESCRIPTION
**Pull request by kevmoo**
Fix string quote for missing environment var

### CODE REVIEW

**Comment by kevmoo** [comment]
See https://github.com/dart-lang/test/actions/runs/4129315716/jobs/7134786864#step:5:54

File: pkgs/firehose/lib/firehose.dart [hunk 1]
```diff
@@ -92,6 +92,15 @@ See also the docs at https://github.com/dart-lang/ecosystem/wiki/Publishing-auto
 
       print('pubspec:');
       var pubspecVersion = package.pubspec.version;
+      if (pubspecVersion == null) {
```
Comment (devoncarew) [inline pkgs/firehose/lib/firehose.dart:95]: It makes sense to handle the NPE here, but the we're also assuming that the target repo should mark that package as `publish_to: none`, right?

**Review by devoncarew — APPROVED** [review]

File: pkgs/firehose/lib/firehose.dart (hunk 1 above)
Comment (kevmoo) [reply pkgs/firehose/lib/firehose.dart:95]: Maybe? We should decide and either error w/ a good message – or warn maybe?

publish_to: none and/or no version – right?

Comment (devoncarew) [reply pkgs/firehose/lib/firehose.dart:95]: The convention this tool is using is to reply on `publish_to: none`; I'd prefer to use that consistently as the 'not intended for publishing signal'.

In previous iterations of the tool I'd had an explicit opt-in signal (`auto-publish: true`) _and_ used `publish_to: none` as a negative signal, but it ended up being more complicated than it was worth - users not exactly knowing which packages would be managed by the publishing automation.

**Pull request merged**